src/qibolab/result.py

```diff
@@ -77,6 +82,16 @@ def __add__(self, data):
         new_res.std = np.append(self.std, data.std)
         return new_res
 
+    @property
+    def average(self):
+        """Perform average over i and q."""
+        return self
+
+    @cached_property
+    def phase_std(self):
+        """Signal phase in radians."""
+        return None
```

Comment:
Even this one sounds more appropriate as an error...

Reply:
Same here